pkg/snapshotmgr/snapshot_manager_test.go

```diff
@@ -29,6 +32,7 @@ import (
 	"os"
 	"strings"
 	"testing"
+	"time"
 )
 
 func TestGetVcConfigSecret(t *testing.T) {
```

Comment:
Please go ahead remove previously existing test cases in this file. Basically, they are not actually test cases. Also, they should not be located in snapshot_manager_test.go